### PR TITLE
[WOR-833] Return landingZoneId for Azure billing projects

### PR DESCRIPTION
**Pull request by cahrens**
Ticket: https://broadworkbench.atlassian.net/browse/WOR-833

To be used by https://github.com/DataBiosphere/terra-ui/pull/3866

---

**PR checklist**

- [ ] Include the JIRA issue number in the PR description and title
- [ ] Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] If you changed anything in `model/`, then you should [publish a new official `rawls-model`](https://github.com/broadinstitute/rawls/blob/develop/README.md#publish-rawls-model) and update `rawls-model` in [Orchestration's dependencies](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/project/Dependencies.scala).
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green, including CI tests
- [ ] **Squash commits and merge** to develop (branches are automatically deleted after merging)
- [ ] Inform other teams of any substantial changes via Slack and/or email

### CODE REVIEW

File: core/src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -5203,7 +5203,7 @@ components:
         cloudPlatform:
           type: string
           description: The cloud platform of the workspace
-          enum: 
+          enum:
```
Comment (cahrens) [inline core/src/main/resources/swagger/api-docs.yaml:5206]: I didn't intentionally change this, but I verified that Swagger still reads correctly with this change.

File: core/src/main/resources/swagger/api-docs.yaml [hunk 2]
```diff
@@ -5892,6 +5892,10 @@ components:
             - GCP
             - AZURE
             - UNKNOWN
+        landingZoneId:
```
Comment (cahrens) [inline core/src/main/resources/swagger/api-docs.yaml:5897]: I suppose I should update orchestration's swagger too? https://firecloud-orchestration.dsde-dev.broadinstitute.org/#/BillingV2/listUserBillingProjectsV2

**Review by aherbst-broad — APPROVED** [review]

**Comment by cahrens** [comment]
jenkins retest

**Review by marctalbott — APPROVED** [review]

File: core/src/main/resources/swagger/api-docs.yaml (hunk 2 above)
Comment (cahrens) [reply core/src/main/resources/swagger/api-docs.yaml:5897]: https://github.com/broadinstitute/firecloud-orchestration/pull/1119

**Pull request merged**